.eslintrc.js

```diff
@@ -252,6 +252,26 @@ module.exports = {
     'no-native-reassign': 2,
     'no-redeclare': 2,
     'no-restricted-globals': [2, 'error', 'event', 'Animation'],
+    'no-restricted-syntax': [
+      2,
+      // Ban all TS features that don't have a direct ECMAScript equivalent.
+      {
+        'selector': 'TSEnumDeclaration',
+        'message': 'Enums are banned.',
+      },
+      {
+        'selector': 'TSModuleDeclaration',
+        'message': 'Namespaces are banned.',
+      },
+      {
+        'selector': 'TSParameterProperty',
+        'message': 'Parameter properties are banned.',
+      },
+      {
+        'selector': 'Decorator',
+        'message': 'Decorators are banned.',
+      },
```

Comment:
WDYT about condensing this?
```suggestion
      {'selector': 'TSEnumDeclaration', 'message': 'Enums are banned.'},
      {'selector': 'TSModuleDeclaration', 'message': 'Namespaces are banned.'},
      {'selector': 'TSParameterProperty', 'message': 'Parameter properties are banned.'},
      {'selector': 'Decorator', 'message': 'Decorators are banned.'},
```

Reply:
horrid 😢

Reply:
`'message': 'No param props pls'`